app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -15,13 +15,17 @@ CONFIG_PM_DEVICE_RUNTIME=y
 CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
 CONFIG_PM_DEVICE_POWER_DOMAIN=y
 CONFIG_PM_POLICY_CUSTOM=y
+CONFIG_PM_STATE_CUSTOM_DATA=y
+CONFIG_ADSP_DYNAMIC_CLOCK_SWITCHING=y
```

Comment:
I strongly suggest to rename it (in zephyr) to CONFIG_ADSP_**IDLE**_DYNAMIC_CLOCK_SWITCHING or something like that to avoid confusion with all kconfigs about clocks and kcps